app/registries/new/controller.js

```diff
@@ -10,6 +10,9 @@ export default Ember.Controller.extend(NewOrEdit, {
 
   actions: {
     selectDriver: function(name) {
+      setTimeout(() => {
+        $('INPUT')[0].focus();
```

Comment:
This needs to be more specific, add a class or look only in `<main>`.  There will likely be an `<input>` in the header soon.